blog.qmd

```diff
@@ -0,0 +1,228 @@
+  message: false
+
+---
+
+```{r}
+#| include: false
+librarian::shelf(glue, gt)
+my_theme <- function(data, ...){
+  data %>%
+    opt_row_striping() %>%
+    opt_all_caps() %>%
+    opt_table_font(
+      font = list(
+        gt::google_font("Lato"),
+        default_fonts()))  %>%
+    tab_style(
+      locations = cells_title(groups = "title"),
+      style     = list(
+        cell_text(weight = "bold", size = 24)
+      )
+    ) %>%
+    #Apply different style to the title
+    tab_style(
+      locations = cells_title(groups = "title"),
+      style     = list(
+        cell_text(weight = "bold", size = 24)
+      )
+    ) %>%
+    tab_options(
+      column_labels.border.top.width = px(5),
+      column_labels.border.top.color = "#FFFFFF",
+      table.border.top.color = "#FFFFFF",
+      table.border.bottom.color = "#FFFFFF",
+      heading.background.color = '#1696d2',
+      data_row.padding = px(5),
+      source_notes.font.size = 16 ,
+      heading.align = "center",
+      row_group.background.color = '#D0D3D4',
+      ...)
+}
+
+```
+
+
+*This blog post is part two of a series on analyzing work-based learning
+opportunities in community colleges. In [part
+one](https://urban-institute.medium.com/how-web-scraping-powered-our-analysis-of-work-based-learning-opportunities-in-community-colleges-3452320dc345),
+we discussed how we used web scraping to gather course descriptions from
+community colleges in Florida. Now, we'll delve into how we analyzed this data
+using the quanteda package in R.*
+
+## Introduction
+
+In our previous post, we detailed our journey of collecting course descriptions
+from Florida's community colleges using web scraping techniques. We successfully
+compiled a comprehensive dataset, but the question remained: how do we make
+sense of this vast amount of text data? Enter quanteda, an R package designed
+for quantitative text analysis.
+
+## Getting Started with Quanteda
+
+Quanteda, short for Quantitative Analysis of Textual Data, is a powerful tool
+for managing and analyzing text data in R. It offers a suite of functions for corpus management, creating document-feature matrices, analyzing keywords, and more. These functions are highly efficient and provide a consistent interface with support for multiple languages. While it operates as a standalone package, it also integrates seamlessly with extensions such as readtext, spacyr, and quanteda.textstats.
+
+To install and load the required packages, we can use the librarian package for convenience
+```{r text-}
+librarian::shelf(tidyverse, # data wrangling
+                 quanteda, # text mining
+                 readtext # read texts and associated document-level meta-data
+)
+```
+
+Next, we load our text data containing course descriptions as well as document-level meta-data using the readtext::readtext() function. We then perform some additional cleaning steps to standardize variable names and focus our analysis on active courses for credit.
```

Comment:
*metadata

Reply:
Is it worth a sentence on what the dataframe below has in it? Just to orient the reader a little more with a couple of important columns or just a broad summary

Reply:
I'm also struggling a little with the dataframes throughout here - I know columns can be widened but it's a little daunting from a UI perspective to try to make sense of what I'm looking at with everything cut off. Any thoughts on ease of interpretation here? Is there a way to change the default widths, since there is already a helpful scroll bar built in?

Reply:
Good points! I edited this to only select the relevant columns and made sure to include column descriptions. I also changed the default widths, let me know if it looks better. I'm thinking we can use a screenshot of that table and put it into word.